packages/cursorless-engine/src/actions/Call.ts

```diff
@@ -1,27 +1,27 @@
 import { Target } from "../typings/target.types";
 import { ensureSingleTarget } from "../util/targetUtils";
 import { Actions } from "./Actions";
-import { Action, ActionReturnValue } from "./actions.types";
+import { ActionReturnValue } from "./actions.types";
 
-export default class Call implements Action {
+export default class Call {
   constructor(private actions: Actions) {
     this.run = this.run.bind(this);
   }
 
-  async run([sources, destinations]: [
-    Target[],
-    Target[],
-  ]): Promise<ActionReturnValue> {
+  async run(
+    sources: Target[],
+    destinations: Target[],
```

Comment:
Hesitant to call something a destination if it's a target. Maybe we go semantically richer and say "callees" and "arguments"?

Reply:
I had the same thought.